templates/composer.yml

```diff
@@ -123,32 +123,6 @@ objects:
             mountPath: "/var/lib/osbuild-composer"
           - name: cache-directory
             mountPath: "/var/cache/osbuild-composer"
-        - image: "quay.io/app-sre/fluentd-hec:1.2.13"
-          name: fluentd-sidecar
-          resources:
-            requests:
-              cpu: "${FLUENTD_CPU_REQUEST}"
-              memory: "${MEMORY_REQUEST}"
-            limits:
-              cpu: "${FLUENTD_CPU_LIMIT}"
-              memory: "${MEMORY_LIMIT}"
-          env:
-            - name: SPLUNK_HEC_TOKEN
-              valueFrom:
-                secretKeyRef:
-                  name: splunk
-                  key: token
-                  optional: false
-            - name: SPLUNK_HEC_URL
-              valueFrom:
-                secretKeyRef:
-                  name: splunk
-                  key: url
```

Comment:
These variables have to be added to the main service container now, no?

Reply:
Yes thanks, I also removed another one that's not needed anymore